fedimint-dbdump/src/main.rs

```diff
@@ -612,6 +616,29 @@ impl<'a> DatabaseDump<'a> {
         self.serialized
             .insert("Client".to_string(), Box::new(client));
     }
+
+    async fn get_database_data(&mut self) {
```

Comment:
It says `_get` but doesn't return anything, has no comment, I'm confused. :D

Reply:
Perhaps "retrieve" is a better name. It just reads it from the database and adds its to its internal map which is essentially an in-memory representation of the database. dump_database then serializes this. I can rename and add some comments.